Content.Server/Chemistry/Events/TransferThroughFaceAttemptEvent.cs

```diff
@@ -0,0 +1,11 @@
+namespace Content.Server.Chemistry.Events;
+
+public sealed class TransferThroughFaceAttemptEvent : CancellableEntityEventArgs
+{
+    public EntityUid Uid { get; }
+
+    public TransferThroughFaceAttemptEvent(EntityUid uid)
+    {
+        Uid = uid;
+    }
+}
```

Comment:
This Uid should be the thing being thrown and should have comments.

Reply:
Is there a way to subscribe to an local event without the need of a component present?